cmd/retrieve_user.go

```diff
@@ -33,6 +33,7 @@ var (
 // retriveUserCmd represents the retriveUser command
 var retriveUserCmd = &cobra.Command{
 	Use:   "get",
+	Aliases: []string{"list"},
```

Comment:
This is a singular resource endpoint, I don't think `list` should be aliased here.